src/lib/normalizeName.js

```diff
@@ -0,0 +1,13 @@
+//normalized function
+const normalizeName = name => {
+  name = name.toLowerCase().trim();
```

Comment:
Love the use of `.trim()` here for catching leading and trailing whitespace. Built-in methods ftw!

A case I'm not seeing handled is if the user (likely accidentally) entered 2 spaces _within_ the name, so something like `Frosted  Flakes`. Not going to consider it blocking, but something to consider in the future when you need to validate input: sometimes users make mistakes and if we can help them by anticipating them it makes our software easier to use!

Reply:
Using a regex like as mentioned [above](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/53/files#r368244248) can more easily address this. If you're looking for what regex, I ask my friend Google something like "Regex for removing extra spaces between words".

I definitely do not have a library of regex patterns memorized 😂

Reply:
I replaced the loop with regex in this commit 
https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/53/commits/9986713fd8f413aad9809ec27769acaa3bd94381
```
const normalizeName = name => {
  return name
    .replace(/[,.;:!-?]/g, '')
    .toLowerCase()
    .replace(/\s+\s/g, ' ')
    .trim();
};
```